e2e-tests/playwright/e2e/github-happy-path.spec.ts

```diff
@@ -89,14 +100,80 @@ test.describe.skip('GitHub Happy path', () => {
     await expect(page.locator(`text=${issuesCountText}`)).toBeVisible();
 
     for (const issue of openIssues.slice(0, 5)) {
-      const issueLocator = page.locator(
-        `text=${issue.title.replace(/\s+/g, ' ')}`,
+      const issueSelector = `text=${issue.title.replace(/\s+/g, ' ')}`;
+      const issueElement = page.locator(issueSelector);
+      await issueElement.scrollIntoViewIfNeeded();
+      await expect(issueElement).toBeVisible();
+    }
+  });
+
+  test('Verify that the Pull/Merge Requests tab renders the 5 most recently updated Open Pull Requests', async () => {
+    await uiHelper.clickTab('Pull/Merge Requests');
+    const openPRs = await BackstageShowcase.getGithubPRs('open');
+    backstageShowcase.verifyPRRows(openPRs, 0, 5);
+  });
+
+  test('Click on the CLOSED filter and verify that the 5 most recently updated Closed PRs are rendered (same with ALL)', async () => {
+    await uiHelper.clickButton('CLOSED', { force: true });
+    const closedPRs = await BackstageShowcase.getGithubPRs('closed');
+
+    for (const closedPR of closedPRs.slice(0, 5)) {
+      await expect(page.locator(`text=${closedPR.title}`)).toBeVisible();
+    }
+  });
+
+  test('Click on the arrows to verify that the next/previous/first/last pages of PRs are loaded', async () => {
+    const allPRs = await BackstageShowcase.getGithubPRs('all', true);
+
+    await uiHelper.clickButton('ALL', { force: true });
+    await backstageShowcase.verifyPRRows(allPRs, 0, 5);
+
+    await backstageShowcase.clickNextPage();
+    await backstageShowcase.verifyPRRows(allPRs, 5, 10);
+
+    const lastPagePRs = Math.floor((allPRs.length - 1) / 5) * 5;
```

Comment:
can it be extract to a method with a documentation explaining what the method is doing, this way we avoid magic numbers